packages/cache/src/SuspenseCache.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import { Suspense } from '@suspensive/react'
+import { TEXT } from '@suspensive/test-utils'
+import { render, screen } from '@testing-library/react'
+import { Cache } from './Cache'
+import { CacheProvider } from './CacheProvider'
+import { SuspenseCache } from './SuspenseCache'
+
+const key = (id: number) => ['key', id] as const
+
+describe('<SuspenseCache />', () => {
+  let cache: Cache
+
+  beforeEach(() => {
+    cache = new Cache()
+  })
+
+  it('should render child component with data from useSuspenseCache hook', async () => {
+    render(
+      <CacheProvider cache={cache}>
+        <Suspense fallback="Loading...">
+          <SuspenseCache options={{ cacheKey: key(1), cacheFn: () => Promise.resolve(TEXT) }}>
```

Comment:
```suggestion
          <SuspenseCache cacheKey={key(1)} cacheFn={() => Promise.resolve(TEXT)}>
```

Reply:
I'm curious about the benefits of separation. If the interface changes, users would have to pass cacheOptions along with key and fn as separate parameters. Wouldn't this be cumbersome for users?

```
const options = cacheOptions({ key: key(1), fn: () => Promise.reslove(TEXT)})

<SuspenseCache cacheKey={options.key} cacheFn={options.fn}>
```

Reply:
I thought user will use cacheOptions like below

```tsx
const options = cacheOptions({ cacheKey: key(1), cacheFn: () => Promise.reslove(TEXT)})

<SuspenseCache {...options} staleTime={200} cacheTime={Infinity}>
```

Reply:
I modified it https://github.com/toss/suspensive/pull/1083/commits/1d2cd64e7176265cf882f4b5f99ba17e2a14b4ac!!